esmvaltool/diag_scripts/aerosols/aero_utils.py

```diff
@@ -0,0 +1,270 @@
+"""Part of the ESMValTool Aerosol diagnostics.
+
+This module contains utility functions commonly used by aerosol
+assessment routines.
+"""
+
+import os
+
+import iris
+import numpy as np
+
+
+class AeroAnsError(Exception):
+    pass
+
+
+# Global variables
+
+# STASH - name mapping file
+
+PATH_AERO = os.path.abspath(os.path.dirname(__file__))
+seasons = ['djf', 'mam', 'jja', 'son']
+nseas = len(seasons)
+mm_dust = 0.100  # Mol mass dust(kg mol-1) - see UKCA_MODE_SETUP: 'mm' arrays
+earth_g = 9.80665  # m/s^2
+
+
+def string_stc(stash):
+    """Convert stashcode to mXXsYYiZZZ format.
+
+    Parameters
+    ----------
+    stash : str
+        UM stashcode with format YYZZZ.
+
+    Returns
+    -------
+    : str
+        UM stashcode with format mXXsYYiZZZ.
+    """
+
+    stash = int(stash)
+
+    if stash < 1 or stash > 99999:
+        raise AeroAnsError('STR_STC: Invalid stashcode ' + str(stash))
+    return 'm01s{0:02d}i{1:03d}'.format(int(stash / 1000), int(stash % 1000))
+
+
```

Comment:
```suggestion
```
It doesn't look like this is used anywhere, and is specific to the native format for 1 model. Can we get rid of it?

Reply:
I've tidied aero_utils.py to what is actually being used for the new diagnostic. The original thinking was include everything for future development, but who knows where that will end up...